probe-rs/tests/source_location.rs

```diff
@@ -80,7 +80,7 @@ fn source_location() {
             Some(SourceLocation {
                 line: Some(*line),
                 column: Some(*col),
-                directory: Some(PathBuf::from("examples")),
+                directory: Some(PathBuf::from("/home/dominik/Coding/microbit/examples")),
```

Comment:
Are you sure you want this hardcoded path? I am unsure how to do it better tho.

Reply:
The whole testing part is a bit unsatisfying, but some path has to be checked here.